apps/frontend/tests/component_tests/MyReviews.cy.tsx

```diff
@@ -0,0 +1,16 @@
+import MyReviews from "../../src/pages/MyReviews";
```

Comment:
`My Reviews` page and tests can be removed. There's no API call to handle getting reviews for a user. We can just add the delete reviews within the view review component

Reply:
Good point, that's now removed!